src/widgets/WidgetMainSmall.vue

```diff
@@ -157,29 +209,19 @@ Indicative monthly payment is a minimum monthly repayment (MMP) of the greater o
   }
 
   &__iconbird {
-    position: absolute;
-    top: 10px;
-    left: 8px;
+    margin: 10px 0 0 8px;
 
     @media (min-width: 430px) {
-      top: 15px;
+      margin: 15px 0 0 8px;
     }
   }
 
-  &__iconcard {
-    display: flex;
-    position: absolute;
-    top: 10px;
-    left: 8px;
-  }
-
   &__close {
-    position: absolute;
-    top: 10px;
-    right: 10px;
```

Comment:
I changed all of these from absolute positioning to static, and solved it with flex, margin and padding instead. It's a bit more maintainable and easier to control spacing.